packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -54,6 +55,7 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
         className={labelWrapperClass}>
         <Label
           {...labelProps}
+          UNSAFE_className={labelClassName}
```

Comment:
hmm maybe I should make Field accept a isQuiet prop

Reply:
That makes sense.